docs/outlook/contextual-outlook-add-ins.md

```diff
@@ -1,143 +1,64 @@
-
 # Contextual Outlook add-ins
 
 Contextual add-ins are Outlook add-ins that activate based on text in a message or appointment. By using contextual add-ins, a user can initiate tasks related to a message without leaving the message itself, which results in an easier and richer user experience.
 
-Contextual add-ins are different from add-ins that involve attachments or that are specific to certain message types. The following are examples of contextual add-ins:
-
+The following are examples of contextual add-ins:
 
 - Choosing an address to open a map of the location.
-    
 - Choosing a string that opens a meeting suggestion add-in.
-    
 - Choosing a phone number to add to your contacts.
-    
-Currently, contextual add-ins are limited to Outlook Web App.
-
->**Note:** Contextual add-ins do not support add-in commands.
 
 ## How to make a contextual add-in
 
-To make a contextual add-in, its add-in manifest must specify the entity or regular expression that can activate it. The entity can be any of the properties in the [Entities](../../reference/outlook/simple-types.md) object. Thus, the add-in manifest must contain a rule of type **ItemHasKnownEntity** or **ItemHasRegularExpressionMatch**. The following example shows how to specify an entity that is a phone number:
-
+A contextual add-in's manifest must include an [ExtensionPoint](../../reference/manifest/extensionpoint.md) element with an `xsi:type` attribute set to `DetectedEntity`. Within the **ExtensionPoint** element, the addin specifies the entity or regular expression that can activate it. If specifying an entity, the entity can be any of the properties in the [Entities](../../reference/outlook/simple-types.md) object *except* `EmailAddress` or `Url`. Thus, the add-in manifest must contain a rule of type **ItemHasKnownEntity** or **ItemHasRegularExpressionMatch**. The following example shows how to specify an entity that is a phone number:
```

Comment:
Well not really though right? You can activate if say a MeetingSuggestion AND an EmailAddress are found, but you can't activate if only an EmailAddress is found.